modules/quality/src/QualitySSIM.cpp

```diff
@@ -0,0 +1,152 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "opencv2/quality/QualitySSIM.hpp"
```

Comment:
"precomp.hpp" first in `.cpp` files.

See contribution/coding style guides on Wiki: https://github.com/opencv/opencv/wiki